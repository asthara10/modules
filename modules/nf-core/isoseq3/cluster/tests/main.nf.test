nextflow_process {

    name "Test Process ISOSEQ3_CLUSTER"
    script "../main.nf"
    config "./nextflow.config"
    process "ISOSEQ3_CLUSTER"

    tag "modules"
    tag "modules_nfcore"
    tag "isoseq3"
    tag "isoseq3/cluster"

    test("test_isoseq3_cluster") {


        when {
            process {
                """
                
                input[0] = [
            [ id:'test' ], // meta map
            file(params.test_data['homo_sapiens']['pacbio']['refine'], checkIfExists: true),
        ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
