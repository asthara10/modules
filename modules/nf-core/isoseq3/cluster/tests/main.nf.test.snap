{
    "versions": {
        "content": [
            [
                
            ]
        ],
        "timestamp": "2023-11-24T14:37:03.430186"
    }
}